collector/lib/CollectorConfig.cpp

```diff
@@ -545,4 +544,38 @@ std::pair<option::ArgStatus, std::string> CollectorConfig::CheckConfiguration(co
   return std::make_pair(ARG_OK, "");
 }
 
+CollectorConfig::ExternalIPsConfig::ExternalIPsConfig(std::optional<sensor::CollectorConfig> runtime_config, bool default_enabled)
```

Comment:
At this point, would it be worth it to split `ExternalIPsConfig` to its own source file and simply add an instance of it to `CollectorConfig`? I think that might be a bit more clear.

Reply:
Actually, never mind, we are simply creating a new copy of this type every time we call the `ExternalIPsConf` method in `CollectorConfig`, just move the `ExternalIPsConfig` to separate .h/.cpp files and leave its construction up to `CollectorConfig`.

Reply:
I moved the ExternalIPsConfig to its own file.